apps/users/models.py

```diff
@@ -484,6 +484,18 @@ def remove_locale(self, locale):
     def get_fallback(cls):
         return cls._meta.get_field('lang')
 
+    @amo.cached_property
+    def mobile_addons(self):
+        return self.mobile_collection().addons.all()
```

Comment:
I'm actually a fan of explicitly referencing `objects` here to make clear what kind of object we're talking about but that's just nitpicking